include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -13453,6 +13457,12 @@
           "extensions" : [ "SPV_INTEL_long_constant_composite" ],
           "version" : "None"
         },
+        {
+          "enumerant" : "OptNoneINTEL",
```

Comment:
Strictly speaking, the capability is called `CapabilityOptNoneINTEL`. I'm not familiar enough with SPIR-V Headers internals to say whether it is a good name or not, because presumably we will get `CapabilityOptNoneINTEL` enumerant in `Capability` enum, which seems repetitive.

Tagging @Fznamznon and @bashbaug here to comment

Reply:
I'm not sure that calling this capability `CapabilityOptNoneINTEL` was a good idea, so that is probably just a bug in spec and it should be called `OptNoneINTEL` instead.

Reply:
Can I merge this PR?

Reply:
> I'm not sure that calling this capability CapabilityOptNoneINTEL was a good idea, so that is probably just a bug in spec and it should be called OptNoneINTEL instead.

Won't it create an ambiguity for the spec, i.e. when you use `OptNoneINTEL`, which particular thing you refer to? Is it Capability or is it Declaration? However, I guess that we can always explicitly state which exact thing we refer to.

@NikitaRudenkoIntel, I think we can go ahead and merge this PR and adjust the spec. @Fznamznon, will you take care about the spec changes?